classes/StaffManager.php

```diff
@@ -672,23 +672,23 @@ public function getBanLengths() {
     }
 
     /** ADMIN PANEL METHODS **/
-    public function getAdminPanelPerms($type): array {
+    public function getAdminPanelPerms(string $type, bool $permission_check = false): array {
```

Comment:
What is the use case for not checking permissions here?

Reply:
Temp fix until I overhaul admin panel. Prior to this, the panel links would display properly.

However on permission check the return would be an empty array (expected) and then just skip this permissions statement. This allowed content admins access to all tools if the link was known. 

This fix tweaks that a bit to actually return the arrays and do a proper permissions check. I'll get it resolved properly post-character launch with the admin panel fixes I want to get done.